contrib/platform/test/com/sun/jna/platform/win32/PowrProfTest.java

```diff
@@ -34,6 +34,11 @@
 public class PowrProfTest extends TestCase {
 
     public void testProcessorPowerInformation() {
+        // MSDN docs for CallNTPowerInformation specify use of GetSystemInfo to
+        // count logical processors for this InformationLevel. The GetSystemInfo
+        // function only counts logical processors on the current Processor
+        // Group. However, the array of results from ProcessorPowerInformation
+        // also are restricted to the current Processor Group.
```

Comment:
(Proactively writing this comment based on what I believe is the situation, awaiting the results of a test from a user with over 64 processors to confirm its accuracy before merging.)

Reply:
Confirmed that this comment is accurate!